src/CMakeLists.txt

```diff
@@ -1080,6 +1080,7 @@ add_custom_target(vstart-base DEPENDS
     crushtool
     rados
     cython_rados
+    cython_rbd
```

Comment:
i use vstart-base on daily basis, and building librbd slows down my build a lot. i agree that the error message from mgr is annoying and i miss the dashboard when librbd binding is missing. but... i don't really want to trade the build time for them.